### PR TITLE
fix(@clayui/css): Update svg4everybody to 2.1.9

### LINKED ISSUE #3506: Update svg4everybody to 2.1.9 to match Portal's
The version of `svg4everybody.js` included with Clay CSS isn't used in Portal, but we should keep them matched.

### PR DESCRIPTION
**Pull request by pat270**
fixes #3506

### CODE REVIEW

**Comment by marcoscv-work** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by marcoscv-work** [comment]
Hey guys!
How this change comes to `liferay-portal`?

**Comment by jbalsas** [comment]
It doesn't

File: packages/clay-css/src/js/svg4everybody.js [hunk 1]
```diff
@@ -1,51 +1,123 @@
 !function(root, factory) {
```
Comment (jbalsas) [inline packages/clay-css/src/js/svg4everybody.js:1]: This, FYI, is another possible license thing we should sort out :D

**Comment by bryceosterhaus** [comment]
merging for todays release

**Pull request merged**